flexmeasures/cli/data_show.py

```diff
@@ -218,4 +221,117 @@ def list_data_sources():
     )
 
 
+@fm_show_data.command("beliefs")
+@with_appcontext
+@click.option(
+    "--sensor-id",
+    "sensor_ids",
+    type=int,
+    required=True,
+    multiple=True,
+    help="ID of sensor(s). This argument can be given multiple times.",
+)
+@click.option(
+    "--from",
+    "start_str",
+    required=True,
+    help="Plot starting at this datetime. Follow up with a timezone-aware datetime in ISO 6801 format.",
+)
+@click.option(
+    "--duration",
+    "duration_str",
+    required=True,
+    help="Duration of the plot, after --from. Follow up with a duration in ISO 6801 format, e.g. PT1H (1 hour) or PT45M (45 minutes).",
+)
+@click.option(
+    "--belief-time-before",
+    "belief_time_before_str",
+    required=False,
+    help="Time at which beliefs had been known. Follow up with a timezone-aware datetime in ISO 6801 format.",
+)
+@click.option(
+    "--source-id",
+    "source_id",
+    required=False,
+    type=int,
+    help="Source of the beliefs (an existing source id).",
+)
+def plot_beliefs(
+    sensor_ids: List[int],
+    start_str: str,
+    duration_str: str,
+    belief_time_before_str: Optional[str],
+    source_id: Optional[int],
+):
+    """
+    Show a simple plot of belief data directly in the terminal.
+    """
+    # handle required params: sensor, start, duration
+    sensor_names: List[str] = []
+    for sensor_id in sensor_ids:
+        sensor: Sensor = Sensor.query.get(sensor_id)
+        if not sensor:
+            click.echo(f"No sensor with id {sensor_id} known.")
+            raise click.Abort
+        sensor_names.append(sensor.name)
+    start = isodate.parse_datetime(start_str)  # TODO: make sure it has a tz
+    duration = isodate.parse_duration(duration_str)
+    # handle belief time
+    belief_time_before = None
+    if belief_time_before_str:
+        belief_time_before = isodate.parse_datetime(belief_time_before_str)
+    # handle source
+    source: DataSource = None
+    if source_id:
+        source = DataSource.query.get(source_id)
+        if not source:
+            click.echo(f"No source with id {source_id} known.")
+            raise click.Abort
+    # query data
+    beliefs_by_sensor = TimedBelief.search(
+        sensors=list(sensor_ids),
+        event_starts_after=start,
+        event_ends_before=start + duration,
+        beliefs_before=belief_time_before,
+        source=source,
```

Comment:
Plotting cases where there are multiple sources or probabilistic beliefs would require, respectively, separate plot traces per source, and confidence intervals. To avoid that here, you can use `one_deterministic_belief_per_event=True`.